main.py

```diff
@@ -38,6 +38,24 @@ async def notify_mesej(message):
         await asyncio.wait([user.send(message) for user in USERS])
 
 
+def wrap_text(message, max_width, indent=24):
+    wrapped_message = str()
+    indent_text = str()
+    message_width = len(message)
+    width = max_width - indent
+
+    for i in range(indent):
+        indent_text += ' '
+
+    for i in range(0, message_width, width):
+        if i > 0:
+            wrapped_message += indent_text
+        wrapped_message += message[i : i + width]
+        if i < message_width - width:
+            wrapped_message += '\n'
+
+    return wrapped_message
+
```

Comment:
Works like a charm. 

![image](https://user-images.githubusercontent.com/49605954/96263234-66663f80-0fe0-11eb-8f46-ddefeda89991.png)

Reply:
Thank you!